src/firefly/java/edu/caltech/ipac/firefly/server/visualize/imageretrieve/ImageFileRetrieverFactory.java

```diff
@@ -1,7 +1,11 @@
 /*
  * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
  */
-package edu.caltech.ipac.firefly.server.visualize;
+
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+package edu.caltech.ipac.firefly.server.visualize.imageretrieve;
```

Comment:
and here... also several more files below all have the same duplicate license info.